modules/nf-core/bcftools/pluginsplit/main.nf

```diff
@@ -15,17 +15,17 @@ process BCFTOOLS_PLUGINSPLIT {
     path(targets)
 
     output:
-    tuple val(meta), path("*.{vcf,vcf.gz,bcf,bcf.gz}"), emit: vcf
-    tuple val(meta), path("*.tbi")                    , emit: tbi, optional: true
-    tuple val(meta), path("*.csi")                    , emit: csi, optional: true
-    path "versions.yml"                               , emit: versions
+    tuple val(meta), path("*/*.{vcf,vcf.gz,bcf,bcf.gz}"), emit: vcf
+    tuple val(meta), path("*/*.tbi")                    , emit: tbi, optional: true
+    tuple val(meta), path("*/*.csi")                    , emit: csi, optional: true
+    path "versions.yml"                                 , emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
     def args = task.ext.args ?: ''
-    def prefix = task.ext.prefix ?: "${meta.id}"
+    def prefix = task.ext.prefix ?: ""
```

Comment:
Nice

Reply:
Could you perhaps add a comment here to explain why task.ext.prefix is empty by default and not meta.id like (almost) all other modules?